cmd/ansible-operator/main.go

```diff
@@ -29,7 +31,16 @@ func main() {
 	flag.Parse()
 	logf.SetLogger(logf.ZapLogger(false))
 
-	mgr, err := manager.New(config.GetConfigOrDie(), manager.Options{})
+	syncPeriod := time.Duration(60 * time.Second)
+
+	if r, ok := os.LookupEnv("RESYNC_PERIOD"); ok {
```

Comment:
I would like this to be a command line arg. I think command line args are how most control loops specify options.  

Also, I think that we should use https://golang.org/pkg/time/#ParseDuration

This allows for users to be more expressive IMO